### DISCUSSION TITLE
How to fix bot getting username added as suffix in group?

### DISCUSSION THREAD

**Discussion by askfriends** [anasty17/mirror-leech-telegram-bot#1265] [Q&A]
i have added the bot in group, now all commands have been appended with bot username as suffix, So how to fix it?

**Comment by anasty17** [comment]
Suffix not controlled by the code. It's from telegram app!

**Reply by askfriends** [reply]
> Suffix not controlled by the code. It's from telegram app!

thanks bro for ur reply, the bot is working fine in group and all commands too, but only /qbmirror is not working, here is the screenshot
![vps](https://user-images.githubusercontent.com/2517219/205705342-04c697c2-bf22-4d05-98ad-4415cd46c770.png)

**Reply by anasty17** [reply]
Promote the bot!

**Reply by askfriends** [reply]
> Promote the bot!

ahh thanks a ton, it worked!
i have another small querry.
I want to change default download location on vps, so how can i do that?
actually i have two hdds so i want to use other one which is 1tb.
Can u write sample path?

**Reply by anasty17** [reply]
https://stackoverflow.com/questions/24309526/how-to-change-the-docker-image-installation-directory

**Reply by askfriends** [reply]
> https://stackoverflow.com/questions/24309526/how-to-change-the-docker-image-installation-directory

bro qbmirror adds !qB after .mkv, the file looks like `Ghulam-E-Musthafa.1997.1080p.Voot.WEB-DL.AAC2.0.x264- [DDS].mkv.!qB`

**Reply by anasty17** [reply]
For all files ?
If yes then check if u have changed qbittorrent settings.

**Reply by askfriends** [reply]
> For all files ? If yes then check if u have changed qbittorrent settings.

for now im getting this for mkv files only and i havnt touched qb settings

**Reply by anasty17** [reply]
Only this torrent! Try again if same issue then send the torrent i will test by myself.
Another question u have selected specific file from torrent? Ur bot is up to date ?
